README.md

```diff
@@ -26,12 +26,26 @@ git config --local core.hooksPath .github/hook-scripts
 
 ## Running tests
 
-To run unit and infra tests:
+### To run unit and infra tests
```

Comment:
I'm aware that these instructions need to be better, but since I don't know how we want to set up API tests in this repo I didn't want to spend much time on this as I think this might change